api_docs/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 264**
+
+* [`POST /user_uploads`](/api/upload-file): `uri` was renamed
+  to `url`, but remains available as a deprecated alias for
+  backwards-compatibility.
```

Comment:
Please propose this API change formally in the "api design" stream; if we're renaming something in the API, it's always worth discussing whether there's other changes to make in the same at the same time (like maybe it should have been called `foo_url`).

Reply:
OK, I have started the discussion [here](https://chat.zulip.org/#narrow/stream/378-api-design/topic/renaming.20response.20parameter/near/1820743).